---
title: Mutational Dynamics of SARS-CoV-2 in Austria - Report 1

authorLinks: "https://cemm.at/en/research/groups/andreas-bergthaler-group/"
affiliations: "CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences"
date: "May 2020"
dataset: "https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov?d=map&r=region&c=region"
abstract: "This project was launched on 27 March 2020 by CeMM in close collaboration with the Medical University of Vienna.


The aim is to sequence 1,000 viral genomes derived of samples taken from Austrian patients infected with SARS-CoV-2, the virus causing COVID-19. The knowledge we gain will help us learn more about the molecular understanding of the COVID-19 pandemic and SARS-CoV-2, the causative pathogen. The results will integrate Austrian viral genome data into a global map of SARS-CoV-2 mutations, which will help decipher the route of the pathogen from country to country."

---

# [Who are we and what do we do?](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov?animate=2019-12-19,2020-04-24,0,0,15000&d=map&r=region&c=region)

(or a cemm picture instead of the map?)
We are a group of scientists at CeMM in Vienna, Austria.
CeMM is a Research Center for Molecular Medicine of the Austrian Academy of Sciences.
We at CeMM are dedicated to better understand diseases such as cancer, immune disorders or COVID-19.
When there are so many [uncertainties]( https://www.theatlantic.com/health/archive/2020/04/pandemic-confusing-uncertainty/610819/ "Why the Coronavirus Is So Confusing") around COVID-19, our mission is to better understand the evolution and the spread of this disease.

# [What does sequencing the viral genome mean and why is it useful?](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov)

The virion of SARS-CoV-2 coronavirus is a tiny oily bubble that contains the genome of the virus – a long code containing information on how to make new viruses.
This is what viruses are good at: they reprogram our cells to make millions of copies of them.
The code is stored in the form of RNA, and the sequence of four letters: _a_, _c_, _g_ and _u_ determine what kind of proteins will the host cell produce when it translates this almost 30,000 letters long code.


New copies of the virus are just like the original: the code packed in the oily bubble. However, when the cell makes a copy of this long code, it will inevitably make typos. These typos are called __mutations__, and most of them do not change the proteins of the virus.
Instead, they serve as “signatures” for us to follow how does the virus spread.


If a mutated genome is packed into the oily membrane and infects a new host, this host will make copies of the mutated genome. It might again make some mistakes introducing other mutations, that could again further spread in the population.
Sequencing means uncovering the sequence of the letters.
A group of people who acquired the virus from the same source will have similar mutational signature. By sequencing the viral genome derived from the samples of infected people, we can or infer the origin of infections.

# [Tracking SARS-CoV-2](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov?animate=2019-12-19,2020-04-24,0,0,15000&d=map&r=region&c=region)

* The first cases of COVID-19 were described in December 2019[3]. Since then, SARS-CoV-2 spread first to different provinces in China and could eventually establish outbreaks in countries all over the world.

* Sequencing the genomes of viruses isolated from infected patients helps us to understand the evolution of the virus. Moreover, mutations in the viral genome that occur during the spread of SARS-CoV-2 can be used like barcodes to identify connections between different infection clusters and to infer transmission lines.

* Until today, CeMM sequenced viral genomes from 21 Austrian patients. In the following analysis, we compared these samples with 8343 viral genomes from 59 countries on all continents.

# [Phylogenetic analysis reveals an early branching event in SARS-CoV-2 evolution](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov?c=gt-S_614&d=tree,entropy)

* The analysis of mutational signatures in the SARS-CoV-2 genome revealed a major branch point in the SARS-CoV-2 phylogeny.
* Based on our sequence analysis, we conclude that this branching event occured already in the early stages of the pandemic in January 2020 and bases on a mutation in codon 614 of the S protein.
* The viral S protein interacts with ACE2 surface proteins on target cells and mediates viral entry into the cell. However, it is not clear whether this S D614G mutation has an effect on the function of the S protein.

<figure class="image"><img src="https://cemm.at/fileadmin/_processed_/8/b/csm_Illustration-SARS-CoV-2_2_Bobby-Rajesh-Malhotra_CeMM_45a7e025a0.png" alt="SARS CoV-2 docking the ACE2 receptor in human cells."><figcaption>SARS CoV-2 docking the ACE2 receptor in human cells. © Bobby Rajesh Malhotra / CeMM</figcaption>

# [The S D614G mutation is dominant in the early European infection clusters](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov?c=gt-S_614&dmax=2020-02-24&dmin=2020-01-11&d=map&s=Germany/BavPat1/2020)

* However, thanks to SARS-CoV-2 sequencing projects all over the world, we can trace the S D614G back to its origin and learn how the virus spreads.
* Our preliminary data suggest that the first two occurences of the D614G mutation in the United Kingdom and Germany on the 22nd and 28th of January were connected to transmissions from China.

# [The S D614G mutation is dominant in the early European infection clusters](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov?animate=2019-12-19,2020-04-24,0,0,15000&c=gt-S_614&d=map&f_region=Europe)

* We estimate that these two introduction events led quickly to local outbreaks in France, Belgium, Italy, Denmark, Iceland, Russia, Turkey and also Austria.
* In the following weeks, viruses with the S D614G mutation appeared all over Europe and established as the dominant strains in many European countries.
* Moreover, sequence analysis suggests further transmission of this strain to North America and subsequent reintroduction events from the USA to many European countries.

# [Connection of local outbreaks in Austria and other European outbreaks](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov?c=gt-S_614&f_country=Austria&d=tree)

This observation is in line with the results from our mutational analysis of SARS-CoV-2 strains from Austrian patients. Most of the patient samples from Austrian patients show the same S D614G mutation shared by many other European strains.

# [Connection of local outbreaks in Austria and other European outbreaks](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov?f_country=Austria&d=map&r=location)

* A reconstruction of the SARS-CoV-2 phylogeny based on our sequencing data allowed us to infer some transmission lines that led to the introduction of SARS-CoV-2 to Austria.
* Based on this analysis, we estimate introduction events connected to outbreaks in the United Kingdom, Spain, Iceland, France, and Italy.

# [Outlook: Focus on Austria](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov?c=division&f_country=Austria&d=tree,map&r=division)

Together with its collaborators all over Austria, CeMM will keep sequencing SARS-CoV-2 isolates from Austrian patients in order to understand how the virus spread to, within and from Austria and in order to provide the research community with insights into the mutational dynamics of SARS-CoV-2.

<figure class="image"><img src="https://cemm.at/fileadmin/_processed_/f/c/csm_Illustration-SARS-CoV-2_1_Bobby-Rajesh-Malhotra_CeMM_8f76ab32e9.png" alt="SARS CoV-2 docking the ACE2 receptor in human cells."><figcaption>© Bobby Rajesh Malhotra / CeMM</figcaption>

# [Explore the data on your own!](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov?d=map&r=region&c=region)

* You want to explore the data on your own and create your own narratives? Then we can provide you with some useful links!

* How to read phylogenetic trees with nextstrain: <https://nextstrain.org/narratives/trees-background>

* What you can show with narratives: <https://nextstrain.org/narratives/intro-to-narratives>

* How you can write your own narrative: <https://nextstrain.org/docs/narratives/introduction>

* You can find examples of existing narratives here and adjust them to create your own version: <https://github.com/jgenger/nCoV-SARS-2-CeMM/tree/master/examplenarratives>

# [Credits & Acknowledgements](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov?d=map&r=region&c=region)
* We gratefully appreciate the community resources and tools provided by the [Nextstrain project](https://nextstrain.org/)[1] which helped us to conduct our phylogenetic analysis and share our results with you.
* We also thank the [GISAID initiative](https://www.gisaid.org/)[2] for providing access to 8343 SARS-CoV-2 sequences that we could use for this analysis.
* We would like to acknowledge our scientific collaborators from the Center for Virology of the Medical University of Vienna who provided us access to patient samples and the WWTF for providing funding for this project.

# [Citations](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov?d=map&r=region&c=region)

1. Hadfield, J., C. Megill, S. M. Bell, J. Huddleston, B. Potter, C. Callender, P. Sagulenko, T. Bedford, and R. A. Neher. [Nextstrain: real-time tracking of pathogen evolution](https://doi.org/10.1093/bioinformatics/bty407), Bioinformatics 2018

2. Shu, Y., and McCauley, J. (2017). [GISAID: Global initiative on sharing all influenza data – from vision to reality](https://www.eurosurveillance.org/content/10.2807/1560-7917.ES.2017.22.13.30494) Eurosurveillance 22, 2–4.

3. Wu, F., Zhao, S., Yu, B., Chen, Y.-M., Wang, W., Song, Z.-G., Hu, Y., Tao, Z.-W., Tian, J.-H., Pei, Y.-Y., et al. (2020). [A new coronavirus associated with human respiratory disease in China](https://doi.org/10.1038/s41586-020-2008-3), Nature 579, 265–269.
