---
title: Sequencing SARS-CoV-2 genomes isolated from COVID-19 patients in Austria to uncover the route of this pathogen across the globe

authorLinks: "https://cemm.at/en/research/groups/andreas-bergthaler-group/"
affiliations: "CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences"
date: "May 2020"
dataset: "https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/subsampled/8000?d=map&r=region&c=region"
abstract: "This project was launched on 27 March 2020 by CeMM in close collaboration with the Medical University of Vienna.

SARS-CoV-2 is the coronavirus causing COVID-19, the new disease that was first described in December 2019 [3]. Our aim is to sequence 1,000 SARS-CoV-2 genomes isolated from Austrian COVID-19 patients. The Austrian viral genome data will be integrated into a global map of SARS-CoV-2 mutations. Laboratories from all around the world have contributed to this map by submitting data derived from local patients. This collaborative effort will help us learn more about the evolution of SARS-CoV-2, as well as the route of this pathogen from country to country."

---

# [Who are we and what do we do?](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/subsampled/8000)

We are an interdisciplinary group of scientists from the CeMM Research Centre for Molecular Medicine of the Austrian Academy of Sciences in Vienna, Austria.


We at CeMM are dedicated to better understand diseases such as cancer, immune disorders or COVID-19.


When there are so many [uncertainties]( https://www.theatlantic.com/health/archive/2020/04/pandemic-confusing-uncertainty/610819/ "Why the Coronavirus Is So Confusing") around COVID-19, our mission is to help the scientific community better understand the evolution and the spread of SARS-CoV-2, the causative pathogen of COVID-19.

<!-- This is right-side text -->
```auspiceMainDisplayMarkdown
# Research Center for Molecular Medicine
<div width="50%" margin="auto">
<p>
<img width="500px" alt="3D medical animation still shot showing the structure of a coronavirus" src="https://cemm.at/fileadmin/_processed_/9/8/cemm_E01_002.jpg"/>
</p>
<p>
</p>

</div>

```


# [How do mutations appear in the viral genome?](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/subsampled/8000)

* The virion of SARS-CoV-2 coronavirus is an oily bubble that contains the genome of the virus – a long code containing information on how to make new viruses.
This is what viruses are good at: they reprogram our cells to make millions of copies of themselves.
* The code is stored in the form of RNA, and the sequence of four nucleotides represented by the letters _A_, _C_, _G_ and _U_ determine what kind of proteins will the host cell produce when it translates this almost 30,000 "letters" long code.
* New copies of the virus are just like the original: the code packed in the oily bubble. However, when the cell makes a copy of this long code, it will inevitably make "typos". These are called mutations, and most of them do not change the proteins of the virus. Instead, they serve as "signatures" for us to follow how does the virus spread.


<!-- This is right-side text -->
```auspiceMainDisplayMarkdown
# The SARS-Cov-2 virion
<div width="50%" margin="auto">
<p>
<img width="500px" alt="3D medical animation still shot showing the structure of a coronavirus" src="https://www.scientificanimations.com/wp-content/uploads/2020/01/3D-medical-animation-coronavirus-structure.jpg"/>
</p>
<p>
The virion consists of a long RNA that codes all the information required for its own manufacturing, hidden inside of an oily membrane. This membrane is rich in protein protrusions, such as the characteristic "spikes", here shown in pink, giving the virions a crown-like shape, from which coronaviruses take their name. These spikes mediate the entry of the virus into human cells. Image credit: https://www.scientificanimations.com/wiki-images/
</p>
Read more about [how coronavirus mutates and spreads](https://www.nytimes.com/interactive/2020/04/30/science/coronavirus-mutations.html)
</div>

```


# [Sequencing the viral genome to track mutations](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/subsampled/8000)

* When a mutated genome is packed into the oily membrane and infects a new host, this host will make copies of this mutated genome.
* It might again make some mistakes introducing other mutations, that could again further spread in the population.
* By sequencing the viral genome, we bring to light the sequence of the letters, thus unveiling mutations.
* A group of people who acquired the virus from the same source will have a similar mutational signature. By sequencing the genomes of viruses isolated from infected patients, we can identify connections between different infection clusters and infer transmission lines. Moreover, it helps us to understand the evolution of the virus.

```auspiceMainDisplayMarkdown
## Phylogenetic trees can capture accumulating mutations
<div width="50%" margin="auto">
<p>
<img width="700px" alt="cartoon of phylogenetic tree and corresponding alignment, with samples labelled A-E" src="https://github.com/nextstrain/nextstrain.org/raw/master/static-site/content/help/01-general/figures/toy_alignment_tree.png"/>
</p>
<p>
The image above shows four sequenced genomes: A, B, C and D. On the left side of the letters, we see the a phylogenetic tree, while on the right side, the corresponding viral genomes are represented as long lines. Mutations are shown as circles along the genome.


Phylogenetic trees track how the genome of the virus accumulates mutations as time passes.
In the example, the first mutation to appear was the one represented by a dark blue circle. This early mutation is present in all sequenced genomes (as seen on the right side of the letters), however, more recent mutations, such as the red ones, are only present in a subset of genomes.
All genomes containing the same mutations are grouped together, as we infer a closer relationship between them. On the example, A, B and C are grouped together as they all contain the yellow mutation.


Whenever we see a horizontal line, it means a mutation has happened. This is why all horizontal lines have a coloured circle.
In the case of D, no further mutation has happened, that is why no horizontal line leads to it.
Similarly, A and B have identical sequences, thus they are linked with a flat vertical line.

This means that whenever the phylogenetic tree branches, it represents a new mutation that has emerged.

Source of image and further information about reading phylogenetic trees: https://nextstrain.org/narratives/trees-background


</div>

```


# [Tracking SARS-CoV-2](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/subsampled/8000?c=region&d=tree&f_country=Austria)

* Here we see the phylogenetic tree of SARS-CoV-2, constructed based on 8000 viral sequences submitted from all around the world. As we have seen previously, every branch of the tree represents a new mutation.

* The tree can be zoomed in by clicking on a branch. Colours indicate the origin of a specific SARS-CoV-2 sample. Unlike in the example figure, circles we see here are representing samples, not mutations. When hovering you mouse over a circle, sample details apprear.

* Until today, CeMM sequenced viral genomes from 242 Austrian patients. These are now integrated into this global map of SARS-CoV-2 mutations.


# [Focus on Austria](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/subsampled/8000?c=division&f_country=Austria&d=tree,map&r=division)

Together with its collaborators all over Austria, CeMM will keep sequencing SARS-CoV-2 isolates from Austrian patients in order to understand how the virus spread to, within and from Austria and in order to gain better insight into the mutational dynamics of SARS-CoV-2.


<figure class="image"><img src="https://cemm.at/fileadmin/_processed_/f/c/csm_Illustration-SARS-CoV-2_1_Bobby-Rajesh-Malhotra_CeMM_8f76ab32e9.png" alt="SARS CoV-2 docking the ACE2 receptor in human cells."><figcaption>© Bobby Rajesh Malhotra / CeMM</figcaption>


# [Explore the data on your own!](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/subsampled/8000?d=map&r=region&c=region)

* You can explore the data in more details by clicking on "Explore the data yourself" in the upper right corner.
* We can recommend more reading if you want to understand more deeply
  * how SARS-CoV-2 mutates and spreads https://www.nytimes.com/interactive/2020/04/30/science/coronavirus-mutations.html
  * how to read phylogenetic trees https://nextstrain.org/narratives/trees-background
  * what kind of proteins are encoded in the viral genome https://www.nytimes.com/interactive/2020/04/03/science/coronavirus-genome-bad-news-wrapped-in-protein.html


# [Credits & Acknowledgements](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/subsampled/8000?d=map&r=region&c=region)
* We gratefully appreciate the community resources and tools provided by the [Nextstrain project](https://nextstrain.org/)[1] which helped us to conduct our phylogenetic analysis and share our results with you.
* We also thank the [GISAID initiative](https://www.gisaid.org/)[2] for providing access to 8343 SARS-CoV-2 sequences that we could use for this analysis.
* We would like to acknowledge our scientific collaborators from the Center for Virology of the Medical University of Vienna who provided us access to patient samples and the WWTF for providing funding for this project, as well as our colleagues.
  * CeMM: Benedikt Agerer, Christoph Bock, Henrique Colaco, Anita Ender, Jakob-Wendelin Genger, Zsofia Keszei, Alexander Lercher, Jan Laine, Bobby Rajesh Malhotra, Alexandra Popa, Michael Schuster, Martin Senekowitsch, Mark Smyth, Giulio Superti-Furga, Thomas Winkler-Penz
  * University of Veterinary Medicine: Lukas Endler
  * Center for Virology, Med Uni Vienna: Judith Aberle, Stephan Aberle, Elisabeth Puchhammer-Stöckl, Monika Redlberger-Fritz
  * Institute for Virology, Med Uni Innsbruck: Wegene Borena, Dorothee von Laer,
  * Internal Medicine II, Med Uni Innsbruck: Manfred Nairz, Günter Weiss
  * AGES (Austrian Agency for Health and Food Safety): Peter Hufnagl, Daniela Schmid


# [Citations](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/subsampled/8000?d=map&r=region&c=region)

1. Hadfield, J., C. Megill, S. M. Bell, J. Huddleston, B. Potter, C. Callender, P. Sagulenko, T. Bedford, and R. A. Neher. [Nextstrain: real-time tracking of pathogen evolution](https://doi.org/10.1093/bioinformatics/bty407), Bioinformatics 2018

2. Shu, Y., and McCauley, J. (2017). [GISAID: Global initiative on sharing all influenza data – from vision to reality](https://www.eurosurveillance.org/content/10.2807/1560-7917.ES.2017.22.13.30494) Eurosurveillance 22, 2–4.

3. Wu, F., Zhao, S., Yu, B., Chen, Y.-M., Wang, W., Song, Z.-G., Hu, Y., Tao, Z.-W., Tian, J.-H., Pei, Y.-Y., et al. (2020). [A new coronavirus associated with human respiratory disease in China](https://doi.org/10.1038/s41586-020-2008-3), Nature 579, 265–269.
