---
title: Mutational Dynamics of SARS-CoV-2 in Austria - Report 1

authorLinks: "https://cemm.at/en/research/groups/andreas-bergthaler-group/"
affiliations: "CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences"
date: "May 2020"
dataset: "https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov?d=map&r=region&c=region"
abstract: "This project was launched on 27 March 2020 by CeMM in close collaboration with the Medical University of Vienna.


The aim is to sequence 1,000 viral genomes derived of samples taken from Austrian patients infected with SARS-CoV-2, the virus causing COVID-19. The knowledge we gain will help us learn more about the molecular understanding of the COVID-19 pandemic and SARS-CoV-2, the causative pathogen. The results will integrate Austrian viral genome data into a global map of SARS-CoV-2 mutations, which will help decipher the route of the pathogen from country to country."

---

# [Who are we and what do we do?](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov?animate=2019-12-19,2020-04-24,0,0,15000&d=map&r=region&c=region)

or a cemm picture instead of the map?
We are an interdisciplinary group of scientists at CeMM in Vienna, Austria. CeMM is a Research Center for Molecular Medicine of the Austrian Academy of Sciences.


We at CeMM are dedicated to better understand diseases such as cancer, immune disorders or COVID-19.


When there are so many [uncertainties]( https://www.theatlantic.com/health/archive/2020/04/pandemic-confusing-uncertainty/610819/ "Why the Coronavirus Is So Confusing") around COVID-19, our mission is to better understand the evolution and the spread of this disease.

# [What are mutations?](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov)

* The virion of SARS-CoV-2 coronavirus is a tiny oily bubble that contains the genome of the virus – a long code containing information on how to make new viruses.
This is what viruses are good at: they reprogram our cells to make millions of copies of themselves.
* The code is stored in the form of RNA, and the sequence of four nucleotides represented by the letters _a_, _c_, _g_ and _u_ determine what kind of proteins will the host cell produce when it translates this almost 30,000 letters long code.
* New copies of the virus are just like the original: the code packed in the oily bubble. However, when the cell makes a copy of this long code, it will inevitably make typos. These typos are called **mutations**, and most of them do not change the proteins of the virus. Instead, they serve as signatures for us to follow how does the virus spread.

<!-- This is right-side text -->
```auspiceMainDisplayMarkdown
# The SARS-Cov-2 virion
<div width="50%" margin="auto">
<p>
<img width="500px" alt="cartoon showing how transmission tree and phylogenetic tree relate" src="https://upload.wikimedia.org/wikipedia/commons/thumb/e/eb/3D_medical_animation_corona_virus.jpg/1600px-3D_medical_animation_corona_virus.jpg"/>
</p>
<p>
The virion consists of a long RNA that codes all the information required for it's own manufacturing, hidden inside of an oily membrane. This membrane is rich in protein protrusions, such as the characteristic "spikes", here shown in yellow, from which coronaviruses take their name.
</p>

</div>

```


# [What does sequencing the viral genome mean and why is it useful?](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov)

* If a mutated genome is packed into the oily membrane and infects a new host, this host will make copies of this mutated genome.
* It might again make some mistakes introducing other mutations, that could again further spread in the population.
* By sequencing the viral genome, we bring to light the sequence of the letters, thus unveiling mutations.
* A group of people who acquired the virus from the same source will have similar mutational signature. By sequencing the genomes of viruses isolated from infected patients, we can identify connections between different infection clusters and infer transmission lines. Moreover, it helps us understand the evolution of the virus.

```auspiceMainDisplayMarkdown
## Accumulating mutations
<div width="50%" margin="auto">
<p>
<img width="700px" alt="Example phylogeny where all or only a subset of cases are included in the final phylogeny" src="https://github.com/nextstrain/nextstrain.org/raw/master/static-site/content/help/01-general/figures/toy_alignment_tree.png"/>
</p>
<p>
This is a phylogenetic tree, that tracks how the genome of the virus accumulates mutations as time passes. Mutations are represented by coloured circles. On the right we see the viral genomes accommodating these mutations. While the early blue mutation is present in all of the genomes, more recent mutations, such as red ones are only present in a subset of genomes.
All genomes containing the yellow mutation are grouped together, as we infer a closer relationship between them.

</div>

```


# [Tracking SARS-CoV-2](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov?d=tree,entropy&f_country=Austria)

Until today, CeMM sequenced viral genomes from 21 Austrian patients. In the following analysis, we compared these samples with 8343 viral genomes from 59 countries on all continents.


# [Credits & Acknowledgements](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov?d=map&r=region&c=region)
* We gratefully appreciate the community resources and tools provided by the [Nextstrain project](https://nextstrain.org/)[1] which helped us to conduct our phylogenetic analysis and share our results with you.
* We also thank the [GISAID initiative](https://www.gisaid.org/)[2] for providing access to 8343 SARS-CoV-2 sequences that we could use for this analysis.
* We would like to acknowledge our scientific collaborators from the Center for Virology of the Medical University of Vienna who provided us access to patient samples and the WWTF for providing funding for this project.

# [Citations](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov?d=map&r=region&c=region)

1. Hadfield, J., C. Megill, S. M. Bell, J. Huddleston, B. Potter, C. Callender, P. Sagulenko, T. Bedford, and R. A. Neher. [Nextstrain: real-time tracking of pathogen evolution](https://doi.org/10.1093/bioinformatics/bty407), Bioinformatics 2018

2. Shu, Y., and McCauley, J. (2017). [GISAID: Global initiative on sharing all influenza data – from vision to reality](https://www.eurosurveillance.org/content/10.2807/1560-7917.ES.2017.22.13.30494) Eurosurveillance 22, 2–4.

3. Wu, F., Zhao, S., Yu, B., Chen, Y.-M., Wang, W., Song, Z.-G., Hu, Y., Tao, Z.-W., Tian, J.-H., Pei, Y.-Y., et al. (2020). [A new coronavirus associated with human respiratory disease in China](https://doi.org/10.1038/s41586-020-2008-3), Nature 579, 265–269.
