---
title: Viral deep sequencing and contact tracing go hand in hand
authors: "Zsofia Keszei, Jakob-Wendelin Genger, Laura Alvarez, Andreas Bergthaler"
authorLinks: "https://cemm.at/en/research/groups/andreas-bergthaler-group/"
affiliations: "CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences"
date: "July 27, 2020"
dataset: "https://nextstrain.org/community/jgenger/SARS-CoV-2-project/NextstrainAustriav19wIDs?c=region&d=map&f_country=Austria&p=full"
abstract: "In Austria, epidemiologists and biologists work together to decipher the transmission lines of COVID-19. Where contact tracing is impossible, sequencing can help, and where many sequences are identical, contact tracing can shed more light on the origin of an infection.

SARS-CoV-2 is the coronavirus causing COVID-19. Under the project <a href='https://cemm.at/research/sars-cov-2-at/about/' target='_ blank'>Mutational Dynamics of SARS-CoV-2 in Austria</a>, CeMM, in collaboration with the Medical University of Vienna has so far sequenced 305 viral genomes isolated from COVID-19 patients in Austria, and the genomic data has been integrated into a global map of SARS-CoV-2 mutations. This week, we focus on how the joined forces of sequencing and contact tracing can be a weapon against COVID-19.

This narrative reflects the situation of July 27, 2020 and will not be updated."
---

# [About the virus, mutations and phylogenetic trees](https://nextstrain.org/community/jgenger/SARS-CoV-2-project/NextstrainAustriav19wIDs?c=region&d=map&f_country=Austria&p=full)

* A SARS-CoV-2 virus particle consists of a long RNA genome that codes all the information required for its own manufacturing, hidden inside an oily membrane. When copying this code consisting of almost 30,000 nucleotide "letters", typos – mutations – are inevitably introduced.

* Different mutated versions of the virus circulate in the population, and by identifying the mutations in the genome, we can decipher which ones are closely related to each other. With their help, we can draw a phylogenetic tree - a "family tree" of different versions of the virus.

<!-- This is right-side text -->
```auspiceMainDisplayMarkdown
<div width="50%">
<p>
<img width="800px" alt="3D medical animation still shot showing the structure of a coronavirus" src="https://www.scientificanimations.com/wp-content/uploads/2020/01/3D-medical-animation-coronavirus-structure.jpg"/>
</p>
<h1 style="font-size="+1">For an introduction to SARS-CoV-2, viral mutations and phylogenetic trees, check our <a href="https://nextstrain.org/community/narratives/bergthalerlab/SARSCoV2/Introduction_en " target="_ blank">Introductory narrative.</a></h1>
For better user experience, we recommend reading this narrative on a computer rather than a smartphone.

Image credit: <a href="https://www.krisp.org.za/news.php?id=421" target="_ blank">scientificanimations</a>
</div>

```

# [Epidemiological "detectives"](https://nextstrain.org/community/jgenger/SARS-CoV-2-project/NextstrainAustriav19wIDs?branchLabel=clade&c=ClusterID&d=tree&label=clade:Vienna1&m=div&p=full)

* Epidemiologists at <a href="https://www.ages.at" target="_ blank">AGES</a>, the Austrian Agency for Health and Food Safety work with the wittiness of a detective to investigate exact transmission lines of COVID-19 from person to person.

* When a person tests positive for SARS-CoV-2, thorough contact tracing is performed aiming to identify the source of the infection and the possible new successive cases.<a href="https://www.sozialministerium.at/dam/jcr:3cab84f4-126f-46fc-9120-34fcfc463450/Beh%C3%B6rdliche%20Vorgangsweise%20bei%20SARS-CoV-2%20Kontaktpersonen_Kontaktpersonennachverfolgung%20(Stand%2014.04.2020)_FINAL.pdf" target="_ blank">[3]</a>

* As the result, patients can be assigned to transmission clusters: groups of SARS-CoV2 cases with a common origin. Different clusters are represented on the graph by colors.

* However, in some cases, the source of the infection was impossible to identify (grey samples).

* In <a href="https://nextstrain.org/community/narratives/bergthalerlab/SARSCoV2/Generalaudience2_en" target="_ blank">our last narrative</a>, we have shown you an example of major local transmission of the virus concentrated in Vienna.

* In this group of infections, apart from seven patients (grey samples), our collaborators at AGES has assigned all the sequenced samples to two epidemiological clusters: "A" (purple samples) and "AL" (blue samples).

* Hover your mouse over a sample to display more information about it, such as cluster membership.


# [Contact tracing and deep sequencing complement each other](https://nextstrain.org/community/jgenger/SARS-CoV-2-project/NextstrainAustriav19wIDs?branchLabel=clade&c=ClusterID&d=tree&label=clade:Vienna1&m=div&p=full)

* People who get infected by the same source, will have the same version of SARS-CoV-2, with possible additional mutations that accumulate while the virus is replicating in the patients.

* With the help of sequencing, we could confirm that all sequenced members of the cluster "A" are closely related to each other.

* In addition, we were able to establish a connection between the epidemiological cluster "A" and a smaller cluster "AL", linked to a social event.

* However, as many samples contain the same exact version of the virus (all samples that sit on a vertical line), we are not able to infer exact transmission lines between them without the help of contact tracing.

# [Contact tracing and deep sequencing complement each other (continued)](https://nextstrain.org/community/jgenger/SARS-CoV-2-project/NextstrainAustriav19wIDs?branchLabel=clade&c=gt-nuc_20457&d=tree&label=clade:Vienna1&m=div&p=full)

* The tree is now coloured by mutations in the viral genome. As you see, three patients assigned to "AL" had a further mutated version of the virus, shown in yellow.

* The same mutated virus was found in a patient assigned to "A" (CeMM0177), suggesting this patient (or one of her contacts) could be the link between "A" and "AL".

* Furthermore, the six patients in this branch who were not assigned to any cluster can be now linked to cluster "A", and their exposure can be further investigated.

* For instance, how the patient CeMM0187 caught the virus was unclear. Now sequencing data shows a close connection of this sample to the patients who attended the event linked to cluster "AL".


# [Why is it important to know the source of infection?](https://nextstrain.org/community/jgenger/SARS-CoV-2-project/NextstrainAustriav20?branchLabel=clade&c=location&d=tree&f_country=Austria&m=div&p=full)

* In Austria, approximately half of the COVID-19 cases were successfully assigned to a cluster.<a href="https://www.ages.at/service/service-presse/pressemeldungen/epidemiologische-abklaerung-am-beispiel-covid-19/" target="_ blank">[4]</a>

* Most of the clusters were linked to leisure activities and household transmission of the virus.<a href="https://www.ages.at/service/service-presse/pressemeldungen/epidemiologische-abklaerung-am-beispiel-covid-19/" target="_ blank">[4]</a>

* This gives important information on how the virus spreads, a valuable tool for decision makers.

* To provide important insight and complement contact tracing, CeMM will continue to work together with its collaborative universities and the AGES in a continued effort of SARS-CoV-2 sequencing from patient samples in Austria.

<!-- This is right-side text -->
```auspiceMainDisplayMarkdown
<div width="50%">
<p>
<img width="800px" alt="3D medical animation still shot showing the structure of a coronavirus" src="https://www.ilpost.it/wp-content/uploads/2020/05/ospedale-sudafrica-ricostruzione-1.jpg"/>
</p>

A great example is an exhaustive investigation of transmission chains in a hospital in Durban, South Africa.<a href="https://www.krisp.org.za/news.php?id=421" target="_ blank">[5]</a>


Although the first COVID-19 patient (P1) in this hospital was examined in isolation in the triage area, he had to cross another room in the Emergency Department (ED). Here, he possibly infected another patient (P3), that occupied the bed framed in green.


Thanks to the valuable findings this study provided, the layout of the Emergency Department has been changed to prevent similar transmission events.

Image source: KwaZulu‐Natal Research Innovation and Sequencing Platform<a href="https://www.krisp.org.za/news.php?id=421" target="_ blank">[5]</a>
</div>

```

# [Investigate further!](https://nextstrain.org/community/jgenger/SARS-CoV-2-project/NextstrainAustriav20?branchLabel=clade&c=location&d=tree,map&f_country=Austria&m=div&p=full)

* Read the <a href="https://www.ages.at/service/service-presse/pressemeldungen/epidemiologische-abklaerung-am-beispiel-covid-19/" target="_ blank">press release</a> of the Austrian Agency for Health and Food Safety about epidemiological "investigation" and COVID-19 clusters in Austria.

* You can read a summary about inspection of virus transmission in the Augustine's Hospital in Durban, South Africa <a href="https://www.sciencemag.org/news/2020/05/study-tells-remarkable-story-about-covid-19-s-deadly-rampage-through-south-african" target="_ blank">here</a>.

* You can explore the data in more detail by clicking on "**Explore the data yourself**" in the upper right corner. This allows you to
  * Click on a branch to magnify it, and click again to zoom out.
  * Filter for samples taken from different geographical locations by scrolling down and clicking on the name of the location under the map.

<p>
<font size="4">If you want to understands why we sequence SARS-CoV-2 samples and how we draw a phylogenetic tree, explore our other narratives on the <a href="https://cemm.at/research/sars-cov-2-at/visualizations/" target="_ blank">CeMM website</a>!</font>


  # [Credits & Acknowledgements](https://nextstrain.org/community/jgenger/SARS-CoV-2-project/NextstrainAustriav20?d=map&p=full)

  * We gratefully appreciate the community resources and tools provided by the <a href="https://nextstrain.org/" target="_ blank">Nextstrain project</a>[1] which helped us to conduct our phylogenetic analysis and share our results with you.
  * We also thank the <a href="https://www.gisaid.org/" target="_ blank">GISAID initiative</a>[2] for providing access to thousands of SARS-CoV-2 sequences that we were able use for this analysis.
  * We would like to acknowledge our scientific collaborators from the Center for Virology of the Medical University of Vienna, the Institute of Virology of the Medical University of Innsbruck and the Austrian Agency for Health and Food Safety, who provided us access to patient samples, and the Vienna Science and Technology Fund (WWTF) for providing funding for this project.
    * CeMM: Benedikt Agerer, Laura Alvarez, Andreas Bergthaler, Christoph Bock, Henrique Colaço, Anita Ender, Jakob-Wendelin Genger, Zsofia Keszei, Jan Laine, Alexander Lercher, Bobby Rajesh Malhotra, Alexandra Popa, Michael Schuster, Martin Senekowitsch, Mark Smyth, Giulio Superti-Furga, Thomas Winkler-Penz
    * University of Veterinary Medicine: Lukas Endler
    * Center for Virology, Med Uni Vienna: Judith Aberle, Stephan Aberle, Elisabeth Puchhammer-Stöckl, Monika Redlberger-Fritz
    * Institute for Virology, MedUni Innsbruck: Wegene Borena, Dorothee von Laer
    * Internal Medicine II, MedUni Innsbruck: Manfred Nairz, Günter Weiss
    * Department of Medicine IV, Kaiser Franz Josef Hospital, Vienna: Mario Karolyi, Erich Pawelka, Alexander Zoufaly
    * AGES (Austrian Agency for Health and Food Safety): Franz Allerberger, Martin Borkovec, Peter Hufnagl, Sabine Maritschnik, Daniela Schmid
    * Department of Data Sciences, Dana-Farber Cancer Institute, Boston, USA: Franziska Michor, Michael Nicholson
    * Institute for Research in Biomedicine (IRB), Barcelona, Spain: Miguel Grau, Francisco Martinez, Nuria Lopez-Bigas, Oriol Pich
    * Department of Theoretical Chemistry, University of Vienna: Michael T. Wolfinger

# [Citations](https://nextstrain.org/community/jgenger/SARS-CoV-2-project/NextstrainAustriav20?d=map&p=full)

1. Hadfield, J., C. Megill, S. M. Bell, J. Huddleston, B. Potter, C. Callender, P. Sagulenko, T. Bedford, and R. A. Neher. [Nextstrain: real-time tracking of pathogen evolution](https://doi.org/10.1093/bioinformatics/bty407), Bioinformatics. 2018;34(23):4121-4123.

2. Shu, Y., and McCauley, J. [GISAID: Global initiative on sharing all influenza data – from vision to reality](https://www.eurosurveillance.org/content/10.2807/1560-7917.ES.2017.22.13.30494) Euro Surveill. 2017;22(13):30494.

3. <a href="https://www.sozialministerium.at/dam/jcr:3cab84f4-126f-46fc-9120-34fcfc463450/Beh%C3%B6rdliche%20Vorgangsweise%20bei%20SARS-CoV-2%20Kontaktpersonen_Kontaktpersonennachverfolgung%20(Stand%2014.04.2020)_FINAL.pdf" target="_ blank">Official SARS-CoV-2 contact tracing protocol of AGES, as of 15.05.2020</a>, accessed on 16.07.2020

4. [Press release of AGES](https://www.ages.at/service/service-presse/pressemeldungen/epidemiologische-abklaerung-am-beispiel-covid-19/), accessed on 16.07.2020

5. Lessells, R., Moosa, Y., de Oliveira, T. [Report into a nosocomial outbreak of coronavirus disease 2019 (COVID‐19) at Netcare St. Augustine’s Hospital](https://www.krisp.org.za/news.php?id=421), KwaZulu‐Natal Research Innovation and Sequencing Platform (KRISP) 2020.
