---
title: Mutational Dynamics of SARS-CoV-2 in Austria

authorLinks: "https://cemm.at/en/research/groups/andreas-bergthaler-group/"
affiliations: "CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences"
date: "May 2020"
dataset: "https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov?d=map&r=region&c=region"
abstract: "This project was launched on 27 March 2020 by CeMM in close collaboration with the Medical University of Vienna.


The aim is to sequence 1,000 viral genomes derived of samples taken from Austrian patients infected with SARS-CoV-2, the virus causing COVID-19. The knowledge we gain will help us learn more about the molecular understanding of the COVID-19 pandemic and SARS-CoV-2. The results will integrate Austrian viral genome data into a global map of SARS-CoV-2 mutations, which will help decipher the route of the pathogen from country to country."

---

# [Who are we and what do we do?](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov?animate=2019-12-19,2020-04-24,0,0,15000&d=map&r=region&c=region)

or a cemm picture instead of the map? Is this slide even needed? I would like to introduce CeMM to build a brand, but it could be done in the abstract.


We are an interdisciplinary group of scientists at CeMM in Vienna, Austria. CeMM is a Research Center for Molecular Medicine of the Austrian Academy of Sciences.


We at CeMM are dedicated to better understand diseases such as cancer, immune disorders or COVID-19.


When there are so many [uncertainties]( https://www.theatlantic.com/health/archive/2020/04/pandemic-confusing-uncertainty/610819/ "Why the Coronavirus Is So Confusing") around COVID-19, our mission is to better understand the evolution and the spread of this disease.

# [How do mutations appear in the viral genome?](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov)

* The virion of SARS-CoV-2 coronavirus is an oily bubble that contains the genome of the virus – a long code containing information on how to make new viruses.
This is what viruses are good at: they reprogram our cells to make millions of copies of themselves.
* The code is stored in the form of RNA, and the sequence of four nucleotides represented by the letters _a_, _c_, _g_ and _u_ determine what kind of proteins will the host cell produce when it translates this almost 30,000 "letters" long code.
* New copies of the virus are just like the original: the code packed in the oily bubble. However, when the cell makes a copy of this long code, it will inevitably make "typos". These are called mutations, and most of them do not change the proteins of the virus. Instead, they serve as "signatures" for us to follow how does the virus spread.

<!-- This is right-side text -->
```auspiceMainDisplayMarkdown
# The SARS-Cov-2 virion
<div width="50%" margin="auto">
<p>
<img width="500px" alt="3D medical animation still shot showing the structure of a coronavirus" src="https://www.scientificanimations.com/wp-content/uploads/2020/01/3D-medical-animation-coronavirus-structure.jpg"/>
</p>
<p>
The virion consists of a long RNA that codes all the information required for its own manufacturing, hidden inside of an oily membrane. This membrane is rich in protein protrusions, such as the characteristic "spikes", here shown in pink, giving the virions a crown-like shape, from which coronaviruses take their name. These spikes mediate the entry of the virus into human cells. Image credit: https://www.scientificanimations.com/wiki-images/
</p>

</div>

```


# [Sequencing the viral genome to track mutations](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov)

* If a mutated genome is packed into the oily membrane and infects a new host, this host will make copies of this mutated genome.
* It might again make some mistakes introducing other mutations, that could again further spread in the population.
* By sequencing the viral genome, we bring to light the sequence of the letters, thus unveiling mutations.
* A group of people who acquired the virus from the same source will have similar mutational signature. By sequencing the genomes of viruses isolated from infected patients, we can identify connections between different infection clusters and infer transmission lines. Moreover, it helps us understand the evolution of the virus.

```auspiceMainDisplayMarkdown
## Phylogenetic trees can capture accumulating mutations
<div width="50%" margin="auto">
<p>
<img width="700px" alt="cartoon of phylogenetic tree and corresponding alignment, with samples labelled A-E" src="https://github.com/nextstrain/nextstrain.org/raw/master/static-site/content/help/01-general/figures/toy_alignment_tree.png"/>
</p>
<p>
The above image shows four sequenced genomes: A, B, C and D. On the left side of the letters, we see the a phylogenetic tree, while on the right side, the corresponding viral genomes are represented as long lines. Mutations are shown as circles along the genome.


Phylogenetic trees track how the genome of the virus accumulates mutations as time passes.
In the example, the first mutation to appear was the one represented by a blue circle. This early mutation is present in all sequenced genomes, however, more recent mutations, such as the red ones, are only present in a subset of genomes.
All genomes containing the yellow mutation are grouped together, as we infer a closer relationship between them.


Whenever we see a horizontal line, it means a mutation has happened. This is why all horizontal lines have a coloured circle.
In the case of D, no further mutation has happened, that is why no horizontal line leads to it.
Similarly, A and B have identical sequences, thus they are linked with a flat vertical line.

This means that whenever the phylogenetic tree branches, a new mutation has emerged.

Source of image and further information about reading phylogenetic trees: https://nextstrain.org/narratives/trees-background


</div>

```


# [Tracking SARS-CoV-2](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov?c=region&d=tree&f_country=Austria)

* Here we see the phylogenetic tree of SARS-CoV-2, constructed based on all 8343 viral sequences submitted from 59 countries on all continents. As we have seen previously, every branch of the tree represents a new mutation. The tree can be zoomed in by clicking at a branch. Colours represent continents.

* Until today, CeMM sequenced viral genomes from 21 Austrian patients. In the following analysis, we compared these samples with the 8343 submitted genomes from all around the world.


# [Phylogenetic analysis reveals an early branching event in SARS-CoV-2 evolution](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov?c=gt-S_614&d=tree,entropy)

* Our preliminary analysis revealed a major branching event of the phylogenetic tree, that occurred already in the early stages of the pandemic in January 2020.

* The new branch is shown in yellow on the phylogenetic tree, and is hallmarked by a mutation in the part of the genome that contains instructions to make the spike on the surface of the virion: the "S" protein, shown in red in the below image.

* The S protein is of crucial importance to us, as it interacts with the human protein called ACE2.  ACE2 is located on the membrane of target cells and mediates viral entry into the cell. However, it is not yet clear whether this mutation has an effect on the function of the S protein or on its binding to ACE2.

* Based on its position within the region coding for the S protein, this mutation is called S D614G.


<figure class="image"><img src="https://cemm.at/fileadmin/_processed_/8/b/csm_Illustration-SARS-CoV-2_2_Bobby-Rajesh-Malhotra_CeMM_45a7e025a0.png" alt="SARS CoV-2 docking the ACE2 receptor in human cells. The viral S protein in shown in red, while ACE2 is shown in blue"><figcaption>SARS CoV-2 docking the ACE2 receptor in human cells. © Bobby Rajesh Malhotra / CeMM</figcaption>


# [The S D614G mutation is dominant in the early European infection clusters](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov?c=gt-S_614&dmax=2020-02-24&dmin=2020-01-11&d=map&s=Germany/BavPat1/2020)

* Thanks to SARS-CoV-2 sequencing projects all over the world, we can trace the S D614G back to its origin and learn how the virus spreads.
* Our preliminary data suggest that the first two occurences of the D614G mutation in the United Kingdom and Germany on the 22nd and 28th of January were connected to transmissions from China.
* However, such statements should be taken with caution, as there might have been other introduction events to Europe we do not know about because of the lack of available sequences.


# [The S D614G mutation is dominant in the early European infection clusters](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov?animate=2019-12-19,2020-04-24,0,0,15000&c=gt-S_614&d=map&f_region=Europe)

* We estimate that the initial introduction events led quickly to local outbreaks in France, Belgium, Italy, Denmark, Iceland, Russia, Turkey and also Austria.
* In the following weeks, viruses with the S D614G mutation appeared all over Europe and established as the dominant strains in many European countries.
* Moreover, sequence analysis suggests further transmission of this strain to North America and subsequent reintroduction events from the USA to many European countries.

# [Connection of local outbreaks in Austria and other European outbreaks](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov?c=gt-S_614&f_country=Austria&d=tree)

This observation is in line with the results from our mutational analysis of SARS-CoV-2 strains from Austrian patients. Most of the samples from Austrian patients show the same S D614G mutation shared by many other European strains.

# [Connection of local outbreaks in Austria and other European outbreaks](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov?f_country=Austria&d=map&r=location)

* A reconstruction of the SARS-CoV-2 phylogeny based on our sequencing data allowed us to infer some transmission lines that led to the introduction of SARS-CoV-2 to Austria.
* Based on this analysis, we estimate introduction events connected to outbreaks in the United Kingdom, Spain, Iceland, France, and Italy.
* It is important to point out that the direction of these transmissions cannot be predicted with certainty, as the virus might have spread undetected before transmission to another country where its genome was sequenced. Hence it might appear to originate from the country where sequencing occurred.

# [Outlook: Focus on Austria](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov?c=division&f_country=Austria&d=tree,map&r=division)

Together with its collaborators all over Austria, CeMM will keep sequencing SARS-CoV-2 isolates from Austrian patients in order to understand how the virus spread to, within and from Austria and in order to provide the research community with insights into the mutational dynamics of SARS-CoV-2.


<figure class="image"><img src="https://cemm.at/fileadmin/_processed_/f/c/csm_Illustration-SARS-CoV-2_1_Bobby-Rajesh-Malhotra_CeMM_8f76ab32e9.png" alt="SARS CoV-2 docking the ACE2 receptor in human cells."><figcaption>© Bobby Rajesh Malhotra / CeMM</figcaption>


# [Explore the data on your own!](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov?d=map&r=region&c=region)

* You can explore the data in more details by clicking on "Explore the data yourself" in the upper right corner.
* We can recommend more reading if you want to understand more deeply
  * how SARS-CoV-2 mutates and spreads https://www.nytimes.com/interactive/2020/04/30/science/coronavirus-mutations.html
  * how to read phylogenetic trees https://nextstrain.org/narratives/trees-background
  * how the S D614G mutation might effect COVID-19 https://www.theatlantic.com/health/archive/2020/05/coronavirus-strains-transmissible/611239/
  * what kind of proteins are encoded in the viral genome https://www.nytimes.com/interactive/2020/04/03/science/coronavirus-genome-bad-news-wrapped-in-protein.html


# [Credits & Acknowledgements](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov?d=map&r=region&c=region)
* We gratefully appreciate the community resources and tools provided by the [Nextstrain project](https://nextstrain.org/)[1] which helped us to conduct our phylogenetic analysis and share our results with you.
* We also thank the [GISAID initiative](https://www.gisaid.org/)[2] for providing access to 8343 SARS-CoV-2 sequences that we could use for this analysis.
* We would like to acknowledge our scientific collaborators from the Center for Virology of the Medical University of Vienna who provided us access to patient samples and the WWTF for providing funding for this project.

# [Citations](https://nextstrain.org/community/jgenger/nCoV-SARS-2-CeMM/ncov?d=map&r=region&c=region)

1. Hadfield, J., C. Megill, S. M. Bell, J. Huddleston, B. Potter, C. Callender, P. Sagulenko, T. Bedford, and R. A. Neher. [Nextstrain: real-time tracking of pathogen evolution](https://doi.org/10.1093/bioinformatics/bty407), Bioinformatics 2018

2. Shu, Y., and McCauley, J. (2017). [GISAID: Global initiative on sharing all influenza data – from vision to reality](https://www.eurosurveillance.org/content/10.2807/1560-7917.ES.2017.22.13.30494) Eurosurveillance 22, 2–4.
