---
title: SARS-CoV-2 mutations in Austria
authors: "Zsofia Keszei, Jacob-Wendelin Genger, Laura Alvarez, Andreas Bergthaler"
authorLinks: "https://cemm.at/en/research/groups/andreas-bergthaler-group/"
affiliations: "CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences"
date: "June 11 2020"
dataset: "https://nextstrain.org/community/jgenger/SARS-CoV-2-project/NextstrainAustriav19?c=location&d=map&dmin=2020-01-11&f_country=Austria&p=full"
abstract: "SARS-CoV-2 is the coronavirus causing COVID-19.

Under the project [Mutational Dynamics of SARS-CoV-2 in Austria](https://cemm.at/research/sars-cov-2-at/about/), CeMM, in collaboration with the Medical University of Vienna have so far sequenced 318 viral genomes isolated from COVID-10 patients in Austria. The genomic data has been integrated into a global map of SARS-CoV-2 mutations.


This week, we will concentrate on how often mutations appear in the SARS-CoV-2 genome and show mutations that likely emerged in Austria."

---

# [About the virus, mutations and phylogenetic trees](https://nextstrain.org/community/jgenger/SARS-CoV-2-project/OnlyAustrianv19?c=clade_membership&l=clock)

The virion consists of a long RNA genome that codes all the information required for its own manufacturing, hidden inside of an oily membrane. When copying this almost 30'000 "letters" long code, typos - mutations - are inevitably introduced.

<!-- This is right-side text -->
```auspiceMainDisplayMarkdown
<div width="50%">
<p>
<img width="800px" alt="3D medical animation still shot showing the structure of a coronavirus" src="https://www.scientificanimations.com/wp-content/uploads/2020/01/3D-medical-animation-coronavirus-structure.jpg"/>
</p>
<p>
For an introduction to SARS-CoV-2, mutations and phylogenetic trees, check our <a href="https://nextstrain.org/community/narratives/bergthalerlab/SARSCoV2/Introduction_en " target="_ blank">introductory narrative!</a> We recommend reading this narrative on a computer rather than a cell phone.
</p>
<a href="https://www.scientificanimations.com/wiki-images/" target="_ blank">Image credit: scientificanimations</a>
</div>

```

# [Mutations can help us draw a phylogenetic tree](https://nextstrain.org/community/jgenger/SARS-CoV-2-project/NextstrainAustriav19?branchLabel=clade&d=tree&f_region=Africa&label=clade:B4&m=div&p=full)

* There is a chance that a mutation occurs every time the 30'000 nucleotides long RNA genome is copied. As the mutated genome can be further spread in the population, mutations will accumulate over time during a pandemic.

* Similarly to how our DNA genome and that of our parents and siblings are alike, virus genomes that largely contain the same mutations are more closely related than those containing many distinct ones.
Every time a new mutation spreads in the population, we know that these mutated genomes are close relatives.

* The example shows viral genomes isolated in Africa. Remember, whenever we see a new branch emerging from a vertical line, it means a mutation have occurred. Hoover your mouse over the branch to see which nucleotide of the 30'000-nucleotide long code has mutated.

* We see that the genomes UG002 and UG001 from Uganda only differ in one mutation. They are closer relatives than UG017 or OS075-CV12 isolated in Nigeria. We can now speculate that the mutation (C16293T) that gave rise to the branch where UG002 and UG001 belongs, have emerged in Uganda.

* However, we might also be wrong. It is possible that this mutation has emerged in another country, where it was never sequenced, but introduced to Uganda multiple times.


# [Mutational rate of SARS-CoV-2 compared to other viruses](https://nextstrain.org/community/jgenger/SARS-CoV-2-project/OnlyAustrianv19?c=clade_membership&d=tree&l=clock&p=full)

* If we look at the sequences of many viral genomes and see how mutations accumulate over time, we can estimate how quickly mutations occur.

* Based on the 8000 genomes present in this dataset, Nextstrain estimates that approximately 24 mutations occur in this long genome, if one viral lineage is passed along from person to person for a whole year.

* In the 30'000 nucleotides long genome, this means approximately 1 mutation per 1000 nucleotide per year.

* This is in line with a recently published estimate: 0.4 - 0.7 mutations per nucleotides per year[3]

* Other coronaviruses like SARS[4] and MERS[5] mutate in a comparable pace, accumulating approximately 1 mutation/1000 nucleotides/year, while influenza virus changes more than twice as quickly, with an estimated rate of 2.3 mutations/1000 nucleotide/year[6].

# [Mutations in the SARS-CoV-2 genome](https://nextstrain.org/community/jgenger/SARS-CoV-2-project/NextstrainAustriav19?c=location&d=tree&f_country=Australia,Austria&m=div&p=full)

* Ever since the first sequenced SARS-CoV-2 genome was published in January 2020[7], many new mutations have been identified.

* On this phylogenetic tree, the x axis shows the total number of mutations each sampled genome accumulated compared to the root of the tree.

* While most of the genomes isolated in Austria contain less than 10 mutations more than the root of the tree, at the bottom of the graph there is one isolate from Australia that accumulated mutations in 36 different nucleotide positions!


# [SARS-CoV-2 genomes isolated from patients in Austria](https://nextstrain.org/community/jgenger/SARS-CoV-2-project/NextstrainAustriav19?branchLabel=clade&d=tree&f_country=Austria&label=clade:A2a&m=div&p=full)

* On the phylogenetic tree, all Austrian samples are shown in dark blue.

* The samples can be found on many different branches of the tree, meaning that the pandemic in Austria was shaped by many different introduction events of the virus to the country.

* You can also see that some samples are closely related to each other, suggesting local transmission of the virus in Austria.


# [Community transmission within Austria](https://nextstrain.org/community/jgenger/SARS-CoV-2-project/NextstrainAustriav19?branchLabel=clade&c=gt-nuc_12832&d=tree&dmin=2020-02-16&f_region=Europe&m=div&p=full&s=Austria/CeMM0003/2020)

* Let us show you one example of local transmission of the virus: the mutation seen in yellow has most probably emerged in Austria.

* Click on "Explore the data yourself" in the upper right corner, and then on the yellow branch leading to the highlighted sample. You will see all the samples of this branch.

(You can return to this narrative by clicking again on the same button.)

* All samples of the branch are originated from patients in Austria.


# [Community transmission within Austria](https://nextstrain.org/community/jgenger/SARS-CoV-2-project/NextstrainAustriav19?branchLabel=clade&c=gt-nuc_12832&d=map&dmin=2020-02-16&f_region=Europe&m=div&p=full&s=Austria/CeMM0003/2020)

* All of these patients were sampled in Vienna, except for CeMM0347, who was diagnosed in Burgenland. However, even this patient has been to Vienna before getting sick.

* The first sequence of this cluster (CeMM0003) was sampled in February, and the patient has travelled to Italy before.

* This is a unique case of an infection cluster where the origin of the virus can be deducted with certain confidence, and the further transmission of the virus is geographically limited.


# [Explore the data on your own!](https://nextstrain.org/community/jgenger/SARS-CoV-2-project/NextstrainAustriav19?branchLabel=clade&c=gt-nuc_12832&d=tree,map&dmin=2020-02-16&f_region=Europe&m=div&p=full&s=Austria/CeMM0003/2020)

* You can explore the data in more details by clicking on "Explore the data yourself" in the upper right corner.

* Click on a branch to magnify it. Click again to zoom out.

* You can also filter for samples taken from different geographical locations be scrolling down and clicking on the name of the location under the map.

* If you want to gain a deeper understanding on this topic, we recommend the following links
  * <a href="https://www.nytimes.com/interactive/2020/04/30/science/coronavirus-mutations.html" target="_ blank">How SARS-CoV-2 mutates and spreads</a>
  * <a href="https://nextstrain.org/narratives/trees-background" target="_ blank">How to read phylogenetic trees</a>

* <font size="3.5">Explore our other narratives on the <a href="https://cemm.at/research/sars-cov-2-at/visualizations/" target="_ blank">CeMM website</a>!</font>

  # [Credits & Acknowledgements](https://nextstrain.org/community/bergthalerlab/SARSCoV2/NextstrainAustria?d=map&r=region&c=region)

  * We gratefully appreciate the community resources and tools provided by the <a href="https://nextstrain.org/" target="_ blank">Nextstrain project</a> [1] which helped us to conduct our phylogenetic analysis and share our results with you.
  * We also thank the <a href="https://www.gisaid.org/" target="_ blank">GISAID initiative</a> [2] for providing access to thousands of SARS-CoV-2 sequences that we could use for this analysis.
  * We would like to acknowledge our scientific collaborators from the Center for Virology of the Medical University of Vienna, the Institute of Virology of the Medical University of Innsbruck and the Austrian Agency for Health and Food Safety, who provided us access to patient samples, and the Vienna Science and Technology Fund (WWTF) for providing funding for this project.
    * CeMM: Benedikt Agerer, Laura Alvarez, Andreas Bergthaler, Christoph Bock, Henrique Colaco, Anita Ender, Jakob-Wendelin Genger, Zsofia Keszei, Jan Laine, Alexander Lercher, Bobby Rajesh Malhotra, Alexandra Popa, Michael Schuster, Martin Senekowitsch, Mark Smyth, Giulio Superti-Furga, Thomas Winkler-Penz
    * University of Veterinary Medicine: Lukas Endler
    * Center for Virology, Med Uni Vienna: Judith Aberle, Stephan Aberle, Elisabeth Puchhammer-Stöckl, Monika Redlberger-Fritz
    * Institute for Virology, Med Uni Innsbruck: Wegene Borena, Dorothee von Laer,
    * Internal Medicine II, Med Uni Innsbruck: Manfred Nairz, Günter Weiss
    * AGES (Austrian Agency for Health and Food Safety): Peter Hufnagl, Daniela Schmid

# [Citations](https://nextstrain.org/community/bergthalerlab/SARSCoV2/NextstrainAustria?d=map&r=region&c=region)

1. Hadfield, J., C. Megill, S. M. Bell, J. Huddleston, B. Potter, C. Callender, P. Sagulenko, T. Bedford, and R. A. Neher. [Nextstrain: real-time tracking of pathogen evolution](https://doi.org/10.1093/bioinformatics/bty407), Bioinformatics 2018

2. Shu, Y., and McCauley, J. (2017). [GISAID: Global initiative on sharing all influenza data – from vision to reality](https://www.eurosurveillance.org/content/10.2807/1560-7917.ES.2017.22.13.30494) Eurosurveillance 22, 2–4.

3. van Dorp, L. et al.  Infect. Genet. Evol. 104351 (2020) [Emergence of genomic diversity and recurrent mutations in SARS-CoV-2](https://doi.org/10.1016/j.meegid.2020.104351), Infect Genet Evol. 2020;83:104351

4. Zhao Z, Li H, Wu X, et al. [Moderate mutation rate in the SARS coronavirus genome and its implications](https://doi.org/10.1186/1471-2148-4-21) BMC Evol Biol. 2004;4:21.

5. Cotten M, Watson SJ, Zumla AI, et al. [Spread, circulation, and evolution of the Middle East respiratory syndrome coronavirus](https://doi.org/10.1128/mbio.01062-13) mBio. 2014;5(1):e01062-13.

6. Xu X, Cox NJ, Bender CA, Regnery HL, Shaw MW. [Genetic variation in neuraminidase genes of influenza A (H3N2) viruses](https://doi.org/10.1006/viro.1996.0519) Virology. 1996;224(1):175‐183.

7. Lu R, Zhao X, Li J, et al. [Genomic characterisation and epidemiology of 2019 novel coronavirus: implications for virus origins and receptor binding](https://doi.org/10.1016/s0140-6736(20)30251-8) Lancet. 2020;395(10224):565‐574.
