---
title: Sequencing SARS-CoV-2 genomes isolated from COVID-19 patients in Austria to uncover the route of this pathogen across the globe
authors: "Zsofia Keszei, Jakob-Wendelin Genger, Laura Alvarez, Andreas Bergthaler"
authorLinks: "https://cemm.at/en/research/groups/andreas-bergthaler-group/"
affiliations: "CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences"
date: "May 21 2020"
dataset: "https://nextstrain.org/community/bergthalerlab/SARSCoV2/NextstrainAustria?d=map&r=region&c=region"
abstract: "The <a href="https://cemm.at/research/sars-cov-2-at/about/" target="_ blank"> Mutational Dynamics of SARS-CoV-2 in Austria</a> project was launched on 27 March 2020 by CeMM in close collaboration with the Medical University of Vienna.

SARS-CoV-2 is the coronavirus causing COVID-19, the new disease that was first described in December 2019[3]. The aim of this project is to sequence 1,000 SARS-CoV-2 genomes isolated in Austria. So far, viral genomes have been isolated from 237 COVID-19 patients in Austria, sequenced and integrated into a global map of SARS-CoV-2 mutations. Laboratories from all around the world have contributed to this map by submitting data derived from local patients. This collaborative effort will help us learn more about the evolution of SARS-CoV-2, as well as the route of this pathogen from country to country."

---

# [Who are we and what do we do?](https://nextstrain.org/community/bergthalerlab/SARSCoV2/NextstrainAustria?animate=2019-12-13,2020-05-08,0,0,30000&branchLabel=none&d=map&p=full)

CeMM’s mission is to achieve maximum scientific innovation in molecular medicine and improve healthcare. At CeMM, an international and creative team of scientists and medical doctors pursues free-minded basic life science research in a large and vibrant environment of outstanding medical tradition and practice.\
\
CeMM operates in a unique mode of super cooperation, connecting biology with medicine, experiments with computation, discovery with translation, and science with society and the arts. The goal of CeMM is to pioneer the science that nurtures the precise, personalized, predictive and preventive medicine of the future.\
\
Since there are so many <a href="https://www.theatlantic.com/health/archive/2020/04/pandemic-confusing-uncertainty/610819/ " target="_ blank">uncertainties</a> around COVID-19, we aim at helping the scientific community to gain a better understanding of the evolution and the spread of SARS-CoV-2, the causative pathogen of COVID-19.


<!-- This is right-side text -->
```auspiceMainDisplayMarkdown
# CeMM Research Center for Molecular Medicine
<div width="50%">
<p>
<img width="600px" alt="CeMM building Vienna" src="https://cemm.at/fileadmin/_processed_/9/8/cemm_E01_002.jpg"/>
</p>
Photo credit: Iris Ranzinger / CeMM
<p>
</p>

</div>

```

# [The SARS-CoV-2 virion: How do mutations appear in the viral genome?](https://nextstrain.org/community/bergthalerlab/SARSCoV2/NextstrainAustria)

* The virion of SARS-CoV-2 coronavirus is an oily bubble that contains the genome of the virus – a long code containing information on how to make new viruses. This is what viruses are good at: They reprogram our cells to make millions of copies of themselves.
* In SARS-CoV-2 coronavirus, this code is stored in the form of RNA, and the sequence of four nucleotides represented by the letters _A_, _C_, _G_ and _U_ determine what kind of proteins the host cell will produce when it translates this almost 30,000-“letters” long code.
* New copies of the virus are just like the original, the code packed in the oily bubble. However, when the cell makes a copy of this long code, it will inevitably make "typos". These are called mutations, and most of them do not change the proteins of the virus. Instead, they serve as "signatures" for us to follow how the virus spreads.
\
<a href="https://www.nytimes.com/interactive/2020/04/30/science/coronavirus-mutations.html" target="_ blank">Read more about how coronavirus mutates and spreads</a>


<!-- This is right-side text -->
```auspiceMainDisplayMarkdown
<div width="50%">
<p>
<img width="800px" alt="3D medical animation still shot showing the structure of a coronavirus" src="https://www.scientificanimations.com/wp-content/uploads/2020/01/3D-medical-animation-coronavirus-structure.jpg"/>
</p>
<p>
The virion consists of a long RNA that codes all the information required for its own manufacturing, hidden inside of an oily membrane. This membrane is rich in protein protrusions, such as the characteristic "spikes", here shown in pink, giving the virions a crown-like shape, from which coronaviruses take their name. These spikes mediate the entry of the virus into human cells.
</p>
<a href="https://www.scientificanimations.com/wiki-images/" target="_ blank">Image credit: scientificanimations</a>
</div>

```


# [Sequencing the viral genome to track mutations](https://nextstrain.org/community/bergthalerlab/SARSCoV2/NextstrainAustria)

* When a mutated genome is packed into the oily membrane and infects a new host, the host's cells will make copies of this mutated genome.
* It might again make some mistakes introducing other mutations, that could again further spread in the population.
* By sequencing the viral genome, we bring to light the order (sequence) in which the letters are combined, thus unveiling mutations.
* A group of people who acquired the virus from the same source will have a similar mutational signature. By sequencing the genomes of viruses isolated from infected patients, we can identify connections between different infection clusters and trace how the virus was transmitted. Moreover, it helps us to understand the evolution of the virus.

```auspiceMainDisplayMarkdown
## Phylogenetic trees can capture accumulating mutations
<div width="60%">
<p>
<img width="700px" alt="Cartoon of phylogenetic tree and corresponding alignment, with samples labelled A-E" src="https://github.com/nextstrain/nextstrain.org/raw/master/static-site/content/help/01-general/figures/toy_alignment_tree.png"/>
</p>
<p>
The image above shows four sequenced genomes: A, B, C and D. On the left side of the letters, we see the a phylogenetic tree, while on the right side, the corresponding viral genomes are represented as long lines. Mutations are shown as circles along the genome.


Phylogenetic trees track how the genome of the virus accumulates mutations as time passes. The image above shows five sequenced genomes: A, B, C, D and E. On the left side of the letters, we see a phylogenetic tree, while on the right side, the corresponding viral genomes are represented as long lines. Mutations are shown as circles along the genome.


In this example, the first mutation to appear was the one represented by a dark blue circle (on the left side). This early mutation is present in all sequenced genomes (as seen on the right side of the letters). However, more recent mutations, such as the red ones, are only present in a subset of genomes.
All genomes containing the same mutations are grouped together, as we infer a closer relationship between them. On the example, A, B and C are grouped together as they all contain the yellow mutation.


Whenever the phylogenetic tree branches (we see a horizontal line on the left side), it means a new mutation has emerged. This is why all horizontal lines have a coloured circle. In the case of D, no further mutation has happened, that is why no branch leads to it. Similarly, A and B have identical sequences, thus they are linked with a flat vertical line.

Source of image and further reading:
<a href="https://nextstrain.org/narratives/trees-background" target="_ blank">Nextstrain narrative on phylogenetic trees</a>

</div>

```

# [Tracking SARS-CoV-2](https://nextstrain.org/community/bergthalerlab/SARSCoV2/NextstrainAustria?d=tree&f_country=Austria&branchLabel=none&)

* The first cases of COVID-19 were described in December 2019 <a href="https://doi.org/10.1038/s41586-020-2008-3" target="_ blank">[3]</a>. Since then, the causative pathogen, SARS-CoV-2, spread first to different provinces in China and eventually established outbreaks in countries all over the world.

* Sequencing the genomes of viruses isolated from infected patients helps us to understand the evolution of the virus. Moreover, mutations in the viral genome that occur during the spread of SARS-CoV-2 can be used like barcodes to identify connections between different infection clusters and follow how the pathogen spreads from country to country.

* To this day, CeMM has sequenced viral genomes from 237 Austrian patients. These are now integrated into this global map of SARS-CoV-2 mutations (phylogenetic tree).

* On the right side we see the phylogenetic tree of SARS-CoV-2, constructed based on 8,000 viral sequences submitted from all around the world. As we have seen previously, every branch of the tree represents a new mutation.

* The tree can be zoomed in by clicking on a branch. Colours indicate the origin of a specific SARS-CoV-2 sample. Unlike in the previous example figure, the circles we see here represent samples not mutations. When hovering your mouse over a circle, sample details will appear.


# [Explore the data on your own!](https://nextstrain.org/community/bergthalerlab/SARSCoV2/NextstrainAustria?d=map&r=region&c=region)

* You can explore the data in more details by clicking on "Explore the data yourself" in the upper right corner.
* We can recommend more reading if you want to understand more deeply

  * <a href="https://www.nytimes.com/interactive/2020/04/30/science/coronavirus-mutations.html" target="_ blank">How SARS-CoV-2 mutates and spreads</a>
  * <a href="https://nextstrain.org/narratives/trees-background" target="_ blank">How to read phylogenetic trees</a>
  * <a href="https://www.nytimes.com/interactive/2020/04/03/science/coronavirus-genome-bad-news-wrapped-in-protein.html" target="_ blank">What kind of proteins are encoded in the viral genome</a>


# [Credits & Acknowledgements](https://nextstrain.org/community/bergthalerlab/SARSCoV2/NextstrainAustria?d=map&r=region&c=region)

* We gratefully appreciate the community resources and tools provided by the <a href="https://nextstrain.org/" target="_ blank">Nextstrain project</a> [1] which helped us to conduct our phylogenetic analysis and share our results with you.
* We also thank the <a href="https://www.gisaid.org/" target="_ blank">GISAID initiative</a> [2] for providing access to thousands of SARS-CoV-2 sequences that we could use for this analysis.
* We would like to acknowledge our scientific collaborators from the Center for Virology of the Medical University of Vienna, the Institute of Virology of the Medical University of Innsbruck and the Austrian Agency for Health and Food Safety, who provided us access to patient samples, and the Vienna Science and Technology Fund (WWTF) for providing funding for this project.
  * CeMM: Benedikt Agerer, Laura Alvarez, Andreas Bergthaler, Christoph Bock, Henrique Colaco, Anita Ender, Jakob-Wendelin Genger, Zsofia Keszei, Jan Laine, Alexander Lercher, Bobby Rajesh Malhotra, Alexandra Popa, Michael Schuster, Martin Senekowitsch, Mark Smyth, Giulio Superti-Furga, Thomas Winkler-Penz
  * University of Veterinary Medicine: Lukas Endler
  * Center for Virology, Med Uni Vienna: Judith Aberle, Stephan Aberle, Elisabeth Puchhammer-Stöckl, Monika Redlberger-Fritz
  * Institute for Virology, Med Uni Innsbruck: Wegene Borena, Dorothee von Laer,
  * Internal Medicine II, Med Uni Innsbruck: Manfred Nairz, Günter Weiss
  * AGES (Austrian Agency for Health and Food Safety): Peter Hufnagl, Daniela Schmid


# [Citations](https://nextstrain.org/community/bergthalerlab/SARSCoV2/NextstrainAustria?d=map&r=region&c=region)

1. Hadfield, J., C. Megill, S. M. Bell, J. Huddleston, B. Potter, C. Callender, P. Sagulenko, T. Bedford, and R. A. Neher. [Nextstrain: real-time tracking of pathogen evolution](https://doi.org/10.1093/bioinformatics/bty407), Bioinformatics 2018

2. Shu, Y., and McCauley, J. (2017). [GISAID: Global initiative on sharing all influenza data – from vision to reality](https://www.eurosurveillance.org/content/10.2807/1560-7917.ES.2017.22.13.30494) Eurosurveillance 22, 2–4.

3. Wu, F., Zhao, S., Yu, B., Chen, Y.-M., Wang, W., Song, Z.-G., Hu, Y., Tao, Z.-W., Tian, J.-H., Pei, Y.-Y., et al. (2020). [A new coronavirus associated with human respiratory disease in China](https://doi.org/10.1038/s41586-020-2008-3), Nature 579, 265–269.
