---
title: Viral sequencing and epidemology goes hand in hand
authors: "Zsofia Keszei, Jakob-Wendelin Genger, Laura Alvarez, Andreas Bergthaler"
authorLinks: "https://cemm.at/en/research/groups/andreas-bergthaler-group/"
affiliations: "CeMM Research Center for Molecular Medicine of the Austrian Academy of Sciences"
date: "July 11 2020"
dataset: "https://nextstrain.org/community/jgenger/SARS-CoV-2-project/NextstrainAustriav20?c=location&d=map&dmin=2020-01-11&f_country=Austria&p=full"
abstract: "In Austria, epidemologists and biologists work together to decipher the transmission lines of COVID-19. Where contact tracing is impossible, sequencing can help, and where many sequences are identical, contact tracing can shed more light on the origin of an infection.

SARS-CoV-2 is the coronavirus causing COVID-19. Under the project <a href='https://cemm.at/research/sars-cov-2-at/about/' target='_ blank'>Mutational Dynamics of SARS-CoV-2 in Austria</a>, CeMM, in collaboration with the Medical University of Vienna has so far sequenced 305 viral genomes isolated from COVID-10 patients in Austria. The genomic data has been integrated into a global map of SARS-CoV-2 mutations. This week, we focus on how the joined forces of sequencing and contact tracing can be a weapon agains COVID-19."

---

# [About the virus, mutations and phylogenetic trees](https://nextstrain.org/community/jgenger/SARS-CoV-2-project/OnlyAustrianv19?c=clade_membership&l=clock)

A virus particle consists of a long RNA genome that codes all the information required for its own manufacturing, hidden inside an oily membrane. When copying this code consisting of almost 30,000 nucleotide "letters", typos – mutations – are inevitably introduced.

<!-- This is right-side text -->
```auspiceMainDisplayMarkdown
<div width="50%">
<p>
<img width="800px" alt="3D medical animation still shot showing the structure of a coronavirus" src="https://www.scientificanimations.com/wp-content/uploads/2020/01/3D-medical-animation-coronavirus-structure.jpg"/>
</p>
<h1 style="font-size="+1">For an introduction to SARS-CoV-2, viral mutations and phylogenetic trees, check our <a href="https://nextstrain.org/community/narratives/bergthalerlab/SARSCoV2/Introduction_en " target="_ blank">Introductory narrative.</a></h1>
For better user experience, we recommend reading this narrative on a computer rather than a smartphone.

Image credit: <a href="https://www.scientificanimations.com/wiki-images/" target="_ blank">scientificanimations</a>
</div>

```

# [Epidemological detectives](https://nextstrain.org/community/jgenger/SARS-CoV-2-project/NextstrainAustriav20?branchLabel=clade&c=location&d=tree&label=clade:Vienna1&m=div&p=full)

* Epidemologists at AGES, the Austrian Agency for Health and Food Safety work with the wittiness of a detective to investigate exact transmission lines from person to person. include link to their contact tracing protocol? https://www.sozialministerium.at/dam/jcr:3cab84f4-126f-46fc-9120-34fcfc463450/Beh%C3%B6rdliche%20Vorgangsweise%20bei%20SARS-CoV-2%20Kontaktpersonen_Kontaktpersonennachverfolgung%20(Stand%2014.04.2020)_FINAL.pdf

* As the result of their hard work, they could assigned patients to transmission clusters: cases likely having a common origin.

* However, it was impossible to identify the source of the infection of some patients.

* In <a href="https://nextstrain.org/community/narratives/bergthalerlab/SARSCoV2/Generalaudience2_en" target="_ blank">our last narrative</a>, we have showed you an example of major local transmission of the virus concentrated in Vienna.

* In this group of infections, apart from six patients, our collaborators at AGES has assigned all infected people to two epidemological clusters: "A" and "AL".

* Hover your mouse over a sample to display more information about it, such as epidemological clusters membership. (IF IT WILL BE possible!)


# [Contact tracing and sequencing complement each other](https://nextstrain.org/community/jgenger/SARS-CoV-2-project/NextstrainAustriav20?branchLabel=clade&c=location&d=tree&label=clade:Vienna1&m=div&p=full)

* People who get infected by the same source, will have the same version of SARS-CoV-2, with possible additional mutations that accumulated while the virus was replicating in one of the patients.

* With the help of sequencing, we could confirm that all members of the cluster "A" sequenced are closely related to each other.

* In addition, we were able to establish a connection the epidemological cluster "A" to the smaller cluster "AL", linked to a social event.

* We have also sequenced two genomes, from patients assigned to "A" (CeMM0197 and CeMM0217), whose genomes were identical to those linked to the social event, suggesting a possible link between the two groups

* Furthermore, the six patients in this branch who were not assigned to any cluster can be now linked to cluster "A", and their exposure can be further investigated.

* For instance, how the patient CeMM0177 catched the virus was unclear. Now sequencing data shows a close connection of this sample to the event patients of cluster "AL" attanded.


# [Why is it important to know the source of infection?](https://nextstrain.org/community/jgenger/SARS-CoV-2-project/NextstrainAustriav20?branchLabel=clade&c=location&d=tree&label=clade:Vienna1&m=div&p=full)

* Bcs you know how to prevent further infections.

* Example from South Africa!


# [Explore the data on your own!](https://nextstrain.org/community/jgenger/SARS-CoV-2-project/NextstrainAustriav20?branchLabel=clade&d=tree,map&f_country=Austria&m=div&p=full)

* You can explore the data in more detail by clicking on "**Explore the data yourself**" in the upper right corner. We recommend this function, as it allows you to
  * Click on a branch to magnify it, and click again for zooming out.
  * Filter for samples taken from different geographical locations by scrolling down and clicking on the name of the location under the map.

* If you want to gain a deeper understanding of this topic, we recommend the following links
  * <a href="https://www.nytimes.com/interactive/2020/04/30/science/coronavirus-mutations.html" target="_ blank">How SARS-CoV-2 mutates and spreads</a>
  * <a href="https://nextstrain.org/narratives/trees-background" target="_ blank">How to read phylogenetic trees</a>

<p>
<p>
<font size="5">Explore our other narratives on the <a href="https://cemm.at/research/sars-cov-2-at/visualizations/" target="_ blank">CeMM website</a>!</font>


  # [Credits & Acknowledgements](https://nextstrain.org/community/jgenger/SARS-CoV-2-project/NextstrainAustriav20?d=map&p=full)

  * We gratefully appreciate the community resources and tools provided by the <a href="https://nextstrain.org/" target="_ blank">Nextstrain project</a>[1] which helped us to conduct our phylogenetic analysis and share our results with you.
  * We also thank the <a href="https://www.gisaid.org/" target="_ blank">GISAID initiative</a>[2] for providing access to thousands of SARS-CoV-2 sequences that we were able use for this analysis.
  * We would like to acknowledge our scientific collaborators from the Center for Virology of the Medical University of Vienna, the Institute of Virology of the Medical University of Innsbruck and the Austrian Agency for Health and Food Safety, who provided us access to patient samples, and the Vienna Science and Technology Fund (WWTF) for providing funding for this project.
    * CeMM: Benedikt Agerer, Laura Alvarez, Andreas Bergthaler, Christoph Bock, Henrique Colaço, Anita Ender, Jakob-Wendelin Genger, Zsofia Keszei, Jan Laine, Alexander Lercher, Bobby Rajesh Malhotra, Alexandra Popa, Michael Schuster, Martin Senekowitsch, Mark Smyth, Giulio Superti-Furga, Thomas Winkler-Penz
    * University of Veterinary Medicine: Lukas Endler
    * Center for Virology, Med Uni Vienna: Judith Aberle, Stephan Aberle, Elisabeth Puchhammer-Stöckl, Monika Redlberger-Fritz
    * Institute for Virology, MedUni Innsbruck: Wegene Borena, Dorothee von Laer
    * Internal Medicine II, MedUni Innsbruck: Manfred Nairz, Günter Weiss
    * AGES (Austrian Agency for Health and Food Safety): Peter Hufnagl, Daniela Schmid

# [Citations](https://nextstrain.org/community/jgenger/SARS-CoV-2-project/NextstrainAustriav20?d=map&p=full)

1. Hadfield, J., C. Megill, S. M. Bell, J. Huddleston, B. Potter, C. Callender, P. Sagulenko, T. Bedford, and R. A. Neher. [Nextstrain: real-time tracking of pathogen evolution](https://doi.org/10.1093/bioinformatics/bty407), Bioinformatics 2018

2. Shu, Y., and McCauley, J. (2017). [GISAID: Global initiative on sharing all influenza data – from vision to reality](https://www.eurosurveillance.org/content/10.2807/1560-7917.ES.2017.22.13.30494) Eurosurveillance 22, 2–4.

3. van Dorp, L. et al.  Infect. Genet. Evol. 104351 (2020) [Emergence of genomic diversity and recurrent mutations in SARS-CoV-2](https://doi.org/10.1016/j.meegid.2020.104351), Infect Genet Evol. 2020;83:104351

4. Zhao Z, Li H, Wu X, et al. [Moderate mutation rate in the SARS coronavirus genome and its implications](https://doi.org/10.1186/1471-2148-4-21) BMC Evol Biol. 2004;4:21.

5. Cotten M, Watson SJ, Zumla AI, et al. [Spread, circulation, and evolution of the Middle East respiratory syndrome coronavirus](https://doi.org/10.1128/mbio.01062-13) mBio. 2014;5(1):e01062-13.

6. Xu X, Cox NJ, Bender CA, Regnery HL, Shaw MW. [Genetic variation in neuraminidase genes of influenza A (H3N2) viruses](https://doi.org/10.1006/viro.1996.0519) Virology. 1996;224(1):175‐183.

7. Wu F, Zhao S, Yu B, et al. [A new coronavirus associated with human respiratory disease in China](https://doi.org/10.1038/s41586-020-2008-3) Nature. 2020;579(7798):265-269.
